src/sqlancer/mysql/MySQLProvider.java

```diff
@@ -148,7 +140,7 @@ private static int mapActions(MySQLGlobalState globalState, Action a) {
 
     @Override
     public void generateDatabase(MySQLGlobalState globalState) throws Exception {
-        while (globalState.getSchema().getDatabaseTables().size() < Randomly.smallNumber() + 1) {
+        while (globalState.getSchema().getDatabaseTables().size() < Randomly.getNotCachedInteger(1, 6)) {
```

Comment:
6 tables seem like a lot here and might have a detrimental effect on performance. Could we perhaps set the limit depending on the test oracle here?